### ISSUE TITLE
Export User Data to PDF

### ISSUE THREAD

**Issue by senyig20** [techx/quill#111]
Might be a helpful enhancement.

**Comment by YasserDRIF** [comment]
@senyig20 Can you describe the situation where you see this needed
I think that exporting the users info to json is a must, but pdf ?

**Comment by senyig20** [comment]
I agree, but I believe it might be much easier if we had a PDF generation module so that we can give the reports generated to the related groups, i.e. name & school data to the security dept, name & dietary info to the Cartering and such.
